ahcore/utils/io.py

```diff
@@ -273,3 +274,19 @@ def validate_checkpoint_paths(config: DictConfig) -> DictConfig:
         if checkpoint_path and not os.path.isabs(checkpoint_path):
             config.trainer.resume_from_checkpoint = pathlib.Path(hydra.utils.get_original_cwd()) / checkpoint_path
         return config
+
+
+def get_git_hash():
```

Comment:
Yes, then it will return 'None'. That's alright, because you will also have the version in ahcore, which will be in the case of the pip install sufficient.